interfaces/compiler/src/main/java/compiler/interfaces/Scala3Compiler.java

```diff
@@ -43,6 +55,16 @@ public void quietCompile(List<CompilationUnit> units) throws CompilationExceptio
       throw new CompilationException(reporter.allErrors().mkString("\n"));
     }
   }
+  public void compileWithRewrite(List<CompilationUnit> units) throws CompilationException {
```

Comment:
It would be easier to pass a `List<Path>` or a `List<String>` here rather than a `List<CompilationUnit>` (which are supposed to be virtual)

Reply:
ok